### PR TITLE
Change logo styling

### LINKED ISSUE #106: Change logo styling

### PR DESCRIPTION
**Pull request by shadre**
Use a new font for logo: Cookie
Customize a text logo in navbar
Customize a text logo on homepage

Closes #106

_Logo on homepage_:

---

_Logo in navbar_:

### CODE REVIEW

**Comment by nodzy** [comment]
wow, nice!

**Comment by student-tomasz** [comment]
Looks great. Good job! 👏 👏 👏 

I'd only move the logo in the navbar a bit lower to be more vertically centered.

**Comment by shadre** [comment]
Logo lowered a little bit in navbar:

**Comment by agalkiewicz** [comment]
:+1:

File: app/assets/stylesheets/shared/_logo.scss [hunk 1]
```diff
@@ -4,6 +4,7 @@
   color: #9692B2;
   letter-spacing: 2px;
   font-family: "Cookie", cursive, sans-serif;
+  margin-top: 2;
```
Comment (student-tomasz) [inline app/assets/stylesheets/shared/_logo.scss]: 2 what? 2 WHAT???!!!

I guess `2px`, right? ;)

File: app/assets/stylesheets/shared/_fonts.scss [hunk 2]
```diff
@@ -1 +1,2 @@
 @import "https://fonts.googleapis.com/css?family=Open+Sans:300,700&subset=latin-ext";
+@import "https://fonts.googleapis.com/css?family=Cookie";
```
Comment (student-tomasz) [inline app/assets/stylesheets/shared/_fonts.scss]: Load both fonts in a single request like so:

``` css
@import "https://fonts.googleapis.com/css?family=Open+Sans:300,700&subset=latin-ext;Cookie&text=Wizlist";
```

File: app/assets/stylesheets/shared/_logo.scss [hunk 3]
```diff
@@ -0,0 +1,31 @@
+#header-logo {
+  float: left;
+  font-size: 2.7em;
+  color: #9692B2;
+  letter-spacing: 2px;
+  font-family: "Cookie", cursive, sans-serif;
+  margin-top: 2;
```
Comment (student-tomasz) [inline app/assets/stylesheets/shared/_logo.scss]: 2 what? 2 WHAT???!!!

I guess `2px`, right? 😉

**Comment by student-tomasz** [comment]
👍

File: app/views/shared/_navigation.html.erb [hunk 4]
```diff
@@ -1,7 +1,7 @@
 <nav class="navbar navbar-default">
   <div class="container">
     <ul class="nav navbar-nav">
-      <li><%= link_to "Wizlist", root_path, class: "navbar-brand" -%></li>
+      <li><%= link_to "Wizlist", root_path, class: "navbar-brand", id:"header-logo" -%></li>
```
Comment (gotar) [inline app/views/shared/_navigation.html.erb]: - you should use class here, `ID` is reserved for `js`
- missing space after id:

**Comment by gotar** [comment]
good enough you can merge

**Pull request merged**